### PR TITLE
format date in TransactionHistory to have same length

### PR DESCRIPTION
**Pull request by mirecmrozek**
image
Closes https://github.com/vacuumlabs/adalite/issues/203

### CODE REVIEW

File: app/frontend/helpers/formatDate.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const formatDate = (date) => {
+  const timeArray = new Date(date * 1000).toLocaleString().split(',')
+  const numberOfSpaces = 10 - timeArray[0].length
+  const numberOfSpaces2 = 12 - timeArray[1].length
```
Comment (refi93) [inline app/frontend/helpers/formatDate.js]: this may fail for locales different that en-US, let's say the slovak one

Comment (refi93) [reply app/frontend/helpers/formatDate.js]: other solution could be to force en-US locale but I would prefer to use browser's locale for optimal UX

**Comment by refi93** [comment]
btw, my browser has a non-monospace font for sans-serif and it ends up looking like this:
screenshot from 2018-11-09 13-21-31

File: app/frontend/helpers/formatDate.js (hunk 1 above)
Comment (mirecmrozek) [reply app/frontend/helpers/formatDate.js]: Yup, totally agree, and it even looks better.

**Pull request merged**